modules/tensor_mechanics/src/materials/abaqus/AbaqusUmatMaterial.C

```diff
@@ -0,0 +1,260 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "AbaqusUmatMaterial.h"
+#include "Factory.h"
+#include "MooseMesh.h"
+#include "RankTwoTensor.h"
+#include "RankFourTensor.h"
+
+#ifdef LIBMESH_HAVE_DLOPEN
+#include <dlfcn.h>
+#endif
```

Comment:
What if it doesn't? What happens then? Is that being caught properly below?

Reply:
Yes, a moose error is emitted otherwise